### DISCUSSION TITLE
webRcade Android "Version"

### DISCUSSION THREAD

**Discussion by Retro-Devils** [webrcade/webrcade#28] [Ideas]
![Screenshot_20220124-100841_webRcade](https://user-images.githubusercontent.com/88673522/150809184-50c675f4-a4f1-423c-9c1d-6a4dde666f34.jpg)
Hello .
So we ran webRcades play site and feed editor through web2apk . 
Worked fine functions great.  Just loads the two sites

You guys have any interest in this ?

**Comment by Retro-Devils** [comment]
V 1.1 released Roatation Issue Fixed. Please let us know if you guysnare against this or anything like that . 
Check it out here 
https://github.com/Retro-Devils/webRcade/blob/main/
Download Here:
https://github.com/Retro-Devils/webRcade/blob/main/webrcade.apk?raw=true

**Comment by raz0red** [comment]
Are you still maintaining this. I am interested in integrating this into the main project. 
Thanks!

**Reply by Retro-Devils** [reply]
I replied to on wrong spot. 
I am about to update the apk. 
I am more than willing to show yo how to do it, as I see you work on webRcade .

**Comment by Retro-Devils** [comment]
i do mantain it in a way. Its just a app that loads the browser. So its mantained by webRcade

**Reply by braziotis1** [reply]
Hi ! Can you upload an updated webrcade apk ?
Thanks

**Reply by Retro-Devils** [reply]
I can at some point sure. I just used web2exe or similar to convert the website.

**Reply by braziotis1** [reply]
Thanks

**Reply by Retro-Devils** [reply]
Sorry for delay lol. I'll be getting this updated version . I just used a web to apk converter.
